SpecFlowMarsProject/Pages/LanguagePage.cs

```diff
@@ -0,0 +1,99 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using SpecFlowMarsProject.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Policy;
+using System.Text;
+using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+
+namespace SpecFlowMarsProject.Tests
+{
+    internal class LanguagePage : CommonDriver
+    {
+
+        public void AddLanguage(IWebDriver driver, string Language)
```

Comment:
5/
Please implement the POM Pattern, please see the below 
https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html